tests/integration/helpers/cluster.py

```diff
@@ -2056,6 +2056,17 @@ def restart_instance_with_ip_change(self, node, new_ip):
     def restart_service(self, service_name):
         run_and_check(self.base_cmd + ["restart", service_name])
 
+    def restart_container(self, container_name):
+        p = subprocess.Popen(
+            ("docker", "restart", "-s", "9", container_name),
```

Comment:
To avoid that future readers will need to look up what `-s` means (yeah, it _probably_ means "signal" but it is not 100% clear), please write `--signal` out.

Reply:
Nit: by convention in this file, parameters ("docker", "restart" etc.) are written:

```py
(
    "docker",
    "restart",
    [...]
)
```